e2e/_suites/fleet/ingest-manager_test.go

```diff
@@ -36,6 +36,12 @@ const ElasticAgentServiceName = "elastic-agent"
 // FleetProfileName the name of the profile to run the runtime, backend services
 const FleetProfileName = "fleet"
 
+var agentVersionBase = "8.0.0-SNAPSHOT"
+
+// agentVersion is the version of the agent to use
+// It can be overriden by ELASTIC_AGENT_VERSION env var
+var agentVersion = agentVersionBase
+
 // stackVersion is the version of the stack to use
 // It can be overriden by STACK_VERSION env var
 var stackVersion = "8.0.0-SNAPSHOT"
```

Comment:
Should this particular variable matches the `agentVersionBase` as done with `agentVersion`?

Reply:
It's initialised with a default value here: https://github.com/elastic/e2e-testing/pull/426/files/490dff615cb06ba0ae276df3b958403fb7cfc38e#diff-4e9ca5d5986bb705c62581784ed63010d9ad3b449c13a7164930a49b77965c77R73

Maybe we can simply define the variable without value: 
```suggestion
var stackVersion string
```